renderdoc/driver/vulkan/vk_dispatch_defs.h

```diff
@@ -181,6 +181,11 @@ struct VkInstDispatchTable
   PFN_vkCreateStreamDescriptorSurfaceGGP CreateStreamDescriptorSurfaceGGP;
 #endif // VK_USE_PLATFORM_GGP
 
+  // VK_FUCHSIA_imagepipe_surface
+#ifdef VK_USE_PLATFORM_FUCHSIA
+  PFN_vkCreateImagePipeSurfaceFUCHSIA CreateImagePipeSurfaceFUCHSIA;
+#endif // VK_USE_PLATFORM_FUCHSIA
```

Comment:
I don't understand how this is compiling, since the exact same definition exists in the same structure around ~50 lines below this, it should be generating an error about a duplicate member name `CreateImagePipeSurfaceFUCHSIA`.

In any event, as the comment at the top mentions this file is autogenerated so you shouldn't add manual additions like this. It doesn't seem necessary since the definition for this function pointer is already present below.